### PR TITLE
Fix links for 7.5-RC-2 release notes

### LINKED ISSUE #20987: Broken links for 7.5-rc-2 release notes
For the release notes here:
https://docs.gradle.org/7.5-rc-2/release-notes.html

There are three broken links. See the text snippets below (you can ctrl+f for them):
"See the [OutgoingVariantsReport](https://docs.gradle.org/7.5-rc-2/dsl/org.gradle.api.tasks.diagnostics.OutgoingVariantsReport.html) DSL reference for more details."
"See the [ResolvableConfigurations](https://docs.gradle.org/7.5-rc-2/dsl/org.gradle.api.tasks.diagnostics.ResolvableConfigurations.html) DSL reference for more details."
"or [environment variables](https://docs.gradle.org/7.5-rc-2/javadoc//org/gradle/api/provider/ProviderFactory.html#environmentVariablesPrefixedBy-java.lang.String-) with names starting"

### PR DESCRIPTION
**Pull request by alllex**
Fixes #20987

### CODE REVIEW

**Review by tresat — APPROVED** [review]

**Review by ljacomet — APPROVED** [review]
LGTM!

Aside from the annoying supposed change to the inline image. If you figure out a way to make this change disappear, please do

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ The `outgoingVariants` report now provides additional information that allows fu
 This report is useful when determining why a particular variant of this producer project was selected by another consumer project when the producer depends upon the consumer.  Run the report from the producer project, to list every variant built by it (including secondary variants only visible to other local projects).  The output contains the capabilities and attributes present on each variant of the producer, along with other information detailed below.  This output can be compared against the output of the new [`resolvableConfigurations` report](#resolvable-configurations) run in the consumer.
```
Comment (abstratt) [inline subprojects/docs/src/docs/release/notes.md:90]: @alllex Is the change to the image below intended?

Comment (alllex) [reply subprojects/docs/src/docs/release/notes.md:90]: This is not a change to the image, rather a typo fix in the alt text right in the beginning of the line.
From
```
![`outoingVariants` output]
```
to
```
![`outgoingVariants` output]
```

I staged the docs locally and the image renders in the same way as before, so I think I didn't change any image content.

@ljacomet FYI

Comment (ljacomet) [reply subprojects/docs/src/docs/release/notes.md:90]: Thanks, I missed that single letter

**Comment by alllex** [comment]
@bot-gradle test and merge

**Pull request merged**